docs_outdoornav_user_manual/features/watchdogs.mdx

```diff
@@ -0,0 +1,101 @@
+| **Inclination Watchdog**         | A watchdog used to monitor the inclination (roll and pitch) of the UGV. It will trigger if the IMU topic reports an inclination in either the roll or the pitch that exceeds the defined limits.|
+| **Node Status Watchdog**         | A watchdog used to monitor the status of a node. It will trigger is the node crashes or does not start up. |
+| **Nodelet Status Watchdog**      | A watchdog used to monitor the status of a nodelet manager and any nodelets associated with it. It will trigger if either the nodelet manager or the nodelet defined in the parameters crash or fail to start up. |
+| **Num Points Watchdog**          | A watchdog used to monitor the status of a LiDAR device. It is designed to trigger if there are insufficient number of points in a PointCloud2 topic. The primary example for such a watchdog is if a customer has forgotten to remove a protective covering from the LiDAR. |
+| **Odometry Covariance Watchdog** | A watchdog used to monitor reliability of the odometry data that is computed by our system. It will trigger if the any of the odometry covariance thresholds have been exceeded. |
+| **RTK Fix Watchdog**             | A watchdog used to monitor whether or not a GNSS unit has an RTK fix. It will trigger if the device has lost RTK fix for more than the defined timeout. |
+| **Topic Data Watchdog**          | A watchdog used to monitor a ROS topic. It will trigger if no data has been received on the topic for more than the defined timeout. |
+
+
+## Watchdogs Page
+
+To access the watchdog page navigate to the **Menu** &srarr; **Autonomy** &srarr; **Watchdogs**.
+
+<center>
+  <figure>
+    <img
+      src="/img/outdoornav_images/ui_watchdogs_page.png"
+      width="1000"
+    />
+    <figcaption>Watchdogs Page</figcaption>
+  </figure>
+</center>
+
+On this page you will be able to see the history of triggered watchdogs, which includes
+the time of day that it was triggered and the action taken by autonomy to prevent any
+harmful behaviour. Users can clear the history as they see fit to reduce the number of
+entries in the table.
+
+
+### Add Watchdog
+
+To add a watchdog, users can simply click the "Add Watchdog" button on the watchdog page.
+This will open up a pop up. Here, the user can enter the required information related 
+creating a new watchdog. The default parameters that need to be entered are:
+
+- **Watchdog Name**: A semantic name for the watchdog.
+- **Autonomy Action**: The type of action that the autonomy should perform when the watchdog
+is triggered. Options are: "Stop", "Pause", "Warn"
+- **Components**: A list of one or more components of outdoornav that the watchdog will
+be classified under. Examples are: "ros", "navigation", "localization", "obstacle_detection".
+- **Watchdog Type**: Selecting the required watchdog type will then prompt the user
```

Comment:
Unfinished train of thought, probably something like: prompt the user to input the relevant information for that watchdog type.